packages/api/tests/endpoints/test-replays.js

```diff
@@ -19,6 +21,30 @@ const AccessToken = require('../../models/access-tokens');
 let accessTokenModel;
 let jwtAuthToken;
 
+/**
+ * create a source queue
+ *
+ * @param {string} queueNamePrefix - prefix of the queue name
+ * @param {string} visibilityTimeout - visibility timeout for queue messages
+ * @returns {Object} - {queueUrl: <url>} queue created
+ */
+async function createSqsQueues(
```

Comment:
Nit: can't we use this helper from `lib/testUtils`?

Reply:
[`030ed97` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/030ed97f35a8892454a2f46e229a87b2bb447203#diff-df2af8d19c0a84c9334f9bf37ffe602f1e7af349aee3c218576a9d3c6cb4590eR17)